sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -626,7 +626,14 @@ private bool TryGetAttributeProperties(AttributeData attributeData, Location? at
                         }
                         else
                         {
-                            attrProperties[namedArgument.Key] = namedArgument.Value.Value;
+                            if (TryParseValueByType(namedArgument.Value, out object? argValue))
+                            {
+                                attrProperties[namedArgument.Key] = argValue;
+                            }
+                            else
+                            {
+                                // TODO: Log diagnostic error
```

Comment:
Is there a GH issue tracking this todo item?

Reply:
Yes! Forgot to link: https://github.com/Azure/azure-functions-dotnet-worker/issues/1878